kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -687,58 +690,106 @@ define([
          */
         function renderLayout() {
             const uniqueID = html.genId(),
-                events = Events.make(),
-                content = div(
+                events = Events.make();
+            let content = div(
+                {
+                    dataElement: 'kb-log',
+                    class: `${cssBaseClass}__container`,
+                },
+                [
+                    div(
+                        {
+                            class: `${cssBaseClass}__status_container`,
+                        },
+                        [
+                            div({
+                                class: `${cssBaseClass}__status_line`,
+                                dataElement: 'status-line',
+                            }),
+                        ]
+                    ),
+                    div(
+                        {
+                            class: `${cssBaseClass}__logs_container`,
+                            dataElement: 'log-container',
+                        },
+                        [
+                            div(
+                                {
+                                    class: `${cssBaseClass}__logs_title collapsed`,
+                                    role: 'button',
+                                    dataToggle: 'collapse',
+                                    dataTarget: '#' + uniqueID,
+                                    ariaExpanded: 'false',
+                                    ariaControls: uniqueID,
+                                },
+                                [span({}, ['Logs'])]
+                            ),
+                            div(
+                                {
+                                    class: `${cssBaseClass}__logs_content_panel collapse`,
+                                    id: uniqueID,
+                                },
+                                [
+                                    renderControls(events),
+                                    div({
+                                        dataElement: 'log-panel',
+                                        class: logContentClass,
+                                    }),
+                                ]
+                            ),
+                        ]
+                    ),
+                ]
+            );
+
+            if (developerMode) {
```

Comment:
this is an insert -- in developer mode, some extra divs are added with job info and the widget and FSM state.